components/steps/Profile.tsx

```diff
@@ -109,6 +109,24 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
     );
   }
 
+  if (process.browser) {
+    if (!saved && !submitted) {
```

Comment:
I would add another state variable for something like `hasChanged` 

My worry is that people will view the application page, won't write anything, but will be given this message anyways because they haven't clicked save

Expected behavior flows should be:
* I see the application form, I type something in, I accidentally close out the window, but am saved because a popup tells me to click save 
* I see the application form, I don't type anything in, I close out the window, and don't see anything because nothing has changed